flye/2.9.1/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:bionic as app
+
+ARG FLYE_VER="2.9.1"
+
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="1"
+LABEL software="Flye"
+LABEL software.version=${FLYE_VER}
+LABEL description="Fast and accurate de novo assembler for single molecule sequencing reads"
+LABEL website="https://github.com/fenderglass/Flye"
+LABEL license="https://github.com/fenderglass/Flye/blob/flye/LICENSE"
+LABEL maintainer1="Erin Young"
+LABEL maintainer1.email="eriny@utah.gov"
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    make \
+    gcc \
+    g++ \
+    python \
+    python-pip \
+    python-setuptools \
+    zlib1g-dev \
+    wget && \
+    rm -rf /var/lib/apt/lists/* && apt-get autoclean
+
+RUN wget https://github.com/fenderglass/Flye/archive/${FLYE_VER}.tar.gz && \
+    tar -xvf ${FLYE_VER}.tar.gz && \
+    rm ${FLYE_VER}.tar.gz && \
+    cd Flye-${FLYE_VER} && \
+    python setup.py build && \
+    python setup.py install && \
+    mkdir /data
+
+WORKDIR /data
+
+ENV LC_ALL=C
+
+FROM app as test
+
+# from https://github.com/fenderglass/Flye/blob/flye/docs/USAGE.md
+RUN wget https://zenodo.org/record/1172816/files/E.coli_PacBio_40x.fasta && \
+    flye --pacbio-raw E.coli_PacBio_40x.fasta --out-dir out_pacbio --threads 4 && \
```

Comment:
FYI the GH Actions runners only have 2 cpus, I'd recommend bumping these 2 commands (line 42 and 44) down to 2 threads

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
Sorry! I just copied what was in the github repo and didn't really think about it!

Reply:
no worries! appreciate you adding the tests!